src/main/java/com/edmunds/tools/databricks/maven/BaseDatabricksJobMojo.java

```diff
@@ -142,6 +142,11 @@ JobTemplateModel getJobTemplateModel() throws MojoExecutionException {
 
             // [BDD-3114] - we want the current environment, to honor what was passed into the build, and not what was serialized [SAE]
             jobTemplateModel.setEnvironment(environment);
+            if (StringUtils.isBlank(databricksRepo)) {
```

Comment:
@javamonkey79 I admit that these three lines are a bit hacky. If we want to move away from relying on project properties, which I agree with the issue reporter, then to work with existing code this was the only solution I could think of that was relatively easy.